qcengine/programs/rdkit.py

```diff
@@ -25,17 +26,19 @@ class Config(ProgramExecutor.Config):
     def __init__(self, **kwargs):
         super().__init__(**{**self._defaults, **kwargs})
 
+    @staticmethod
+    def found(raise_error: bool=False) -> bool:
+        return which_import("rdkit", return_bool=True, raise_error=raise_error)
+
```

Comment:
I think you want logic more like https://github.com/MolSSI/QCEngine/blob/master/qcengine/programs/psi4.py#L34-L39 . The idea is that sometimes you want to know yes/no found and sometimes you want to find-or-die, and this is the place to consolidate the latter's error msg.